packages/inventory/src/components/table/InventoryTable.js

```diff
@@ -0,0 +1,85 @@
+import React, { Fragment, forwardRef } from 'react';
+import { useSelector } from 'react-redux';
+import EntityTableToolbar from './EntityTableToolbar';
+import { TableToolbar } from '@redhat-cloud-services/frontend-components/components/esm/TableToolbar';
+import InventoryList from './InventoryList';
+import Pagination from './Pagination';
+
+/**
+ * This component is used to combine all essential components together:
+ *   * EntityTableToolbar - to control top toolbar.
+ *   * InventoryList - to allow consumers to change data from outside and contains actual inventory table.
+ *   * Pagination - bottom pagination.
+ * It also calculates pagination and sortBy from props or from store if consumer passed items or not.
+ */
+const InventoryTable = forwardRef(({
+    onRefresh,
+    children,
+    inventoryRef,
+    items,
+    total,
+    page,
+    perPage,
+    filters,
+    showTags,
+    sortBy: propsSortBy,
+    ...props
+}, ref) => {
+    const hasItems = Boolean(items);
+    let pagination = {
```

Comment:
should be const. I would be also a bit happier if the selectors would be used outside of the property declaration. They will be called on each render like this since the variable is always instantiated.